ansible/roles/software/ckan/security-fix/tasks/main.yml

```diff
@@ -10,3 +10,21 @@
   with_items:
     - /etc/ckan
     - "{{ project_source_new_code_path }}"
+
+- name: Fix 'Content-Cache' header
+  replace:
+    dest: "{{ project_source_new_code_path }}/src/ckan/ckan/lib/base.py"
+    regexp: ' = "private"'
```

Comment:
that is possible. `lineinfile` updates only 1 last found regex match though, not necessary the correct one )